### PR TITLE
change 'ImagePullPolicy: Always' description to reflect source code

### PR DESCRIPTION
**Pull request by adamkgray**

### CODE REVIEW

File: content/en/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -67,12 +67,8 @@ these values have:
...
 
 `Always`
-: every time the kubelet launches a container, the kubelet queries the container
-  image registry to resolve the name to an image
-  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a
-  container image with that exact digest cached locally, the kubelet uses its cached
-  image; otherwise, the kubelet pulls the image with the resolved digest,
-  and uses that image to launch the container.
+: every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image
+  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). The kubelet then pulls the image with the resolved digest, and uses that image to launch the container.
```
Comment (tengqm) [inline content/en/docs/concepts/containers/images.md:71]: pls wrap the long lines as we did before.

File: content/zh-cn/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -110,12 +110,8 @@ these values have:
...
 
 `Always`
-: every time the kubelet launches a container, the kubelet queries the container
-  image registry to resolve the name to an image
-  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a
-  container image with that exact digest cached locally, the kubelet uses its cached
-  image; otherwise, the kubelet pulls the image with the resolved digest,
-  and uses that image to launch the container.
+: every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image
+  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). The kubelet then pulls the image with the resolved digest, and uses that image to launch the container.
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/containers/images.md:114]: Please don't update the zh-cn localization. The commented text needs a translation and the localization team has its own pace on this.

File: content/en/docs/concepts/containers/images.md (hunk 1 above)
Comment (sftim) [inline content/en/docs/concepts/containers/images.md:71]: It looks to me as if the kubelet instructs the container runtime to pull that image. In that case, the cacheing behavior also depends on your chosen container runtime.

**Review by sftim — COMMENTED** [review]
/sig node

**Comment by sftim** [comment]
@adamkgray this PR will be easier to review if you fill in a description of the changes, including your motivation for suggesting them.

**Comment by sftim** [comment]
@adamkgray please write a description for this PR that explains _why_ we should make this change. That'll help a lot with the review process.

**Comment by tengqm** [comment]
/close
No response from the author for over one month.
Feel free to reopen.

**Pull request closed without merging**